server/balancer_worker.go

```diff
@@ -275,3 +274,46 @@ func (bw *balancerWorker) storeScores(store *storeInfo) []int {
 
 	return scores
 }
+
+func collectOperatorMetrics(bop *balanceOperator) map[string]uint64 {
+	metrics := make(map[string]uint64)
+	prefix := ""
+	switch bop.Type {
+	case adminOP:
+		prefix = "admin"
+	case replicaOP:
+		prefix = "replica"
+	case balanceOP:
+		prefix = "balance"
+	}
+	for _, op := range bop.Ops {
+		if _, ok := op.(*onceOperator); ok {
+			op = op.(*onceOperator)
+		}
+		switch o := op.(type) {
+		case *changePeerOperator:
+			metrics[prefix+o.Name]++
```

Comment:
Yep, like "balance_add_peer" or "balance_remove_peer", but I forgot to add a underline...